src/renderer/components/watch-video-info/watch-video-info.scss

```diff
@@ -70,31 +70,43 @@
 .videoMetrics {
   font-size: 14px;
   color: var(--tertiary-text-color);
-
```

Comment:
Why removing the blank line

Reply:
i mistakenly removed it, but it should be back now